unable to follow directions to install Crosstool-NG on OS X

**_Reported by Andrew Boie:_**

https://www.zephyrproject.org/doc/getting_started/installation_mac.html

At the step where we invoke CrossTool-NG to build the compiler, I am getting:

```
[INFO ]  Installing cross-gdb: done in 137.00s (at 24:44)
[INFO ]  =================================================================
[INFO ]  Finalizing the toolchain's directory
[INFO ]    Stripping all toolchain executables
[EXTRA]    Creating toolchain aliases
[EXTRA]    Removing installed documentation
stat: illegal option -- c
usage: stat [-FlLnqrsx] [-f format] [-t timefmt] [file ...]
[ERROR]  
[ERROR]  >>
[ERROR]  >>  Build failed in step 'Finalizing the toolchain's directory'
[ERROR]  >>        called in step '(top-level)'
[ERROR]  >>
[ERROR]  >>  Error happened in: CT_DoForceRmdir[scripts/functions@577]
[ERROR]  >>        called from: do_finish[scripts/build/internals.sh@141]
[ERROR]  >>        called from: main[scripts/crosstool-NG.sh@653]
[ERROR]  >>
[ERROR]  >>  For more info on this error, look at the file: 'build.log'
[ERROR]  >>  There is a list of known issues, some with workarounds, in:
[ERROR]  >>      '/usr/local/share/doc/crosstool-ng/crosstool-ng-1.23.0/B - Known issues.txt'
[ERROR]  >>
[ERROR]  >>  If you feel this is a bug in crosstool-NG, report it at:
[ERROR]  >>      https://github.com/crosstool-ng/crosstool-ng/issues/
[ERROR]  >>
[ERROR]  >>  Make sure your report includes all the information pertinent to this issue.
[ERROR]  >>  Read the bug reporting guidelines here:
[ERROR]  >>      http://crosstool-ng.github.io/support/
[ERROR]  
[ERROR]  (elapsed: 24:45.00)
make: *** [build] Error 1
andrew@magrathea:/Volumes/CrossToolNG/build (2)
```

(Imported from Jira ZEP-2247)

--- a/boards/x86/galileo/doc/galileo.rst
+++ b/boards/x86/galileo/doc/galileo.rst
@@ -217,9 +217,10 @@ GRUB.
    .. code-block:: console
 
      $ cd $ZEPHYR_BASE
-     $ ./scripts/build_grub.sh
+     $ ./boards/x86/galileo/support/build_grub.sh
 
-#. Find the binary at :file:`$ZEPHYR_BASE/scripts/grub/bin/grub.efi`.
+#. Find the binary at
+   :file:`$ZEPHYR_BASE/./boards/x86/galileo/support/grub/bin/grub.efi`.
 
 
 

--- a/doc/getting_started/installation_mac.rst
+++ b/doc/getting_started/installation_mac.rst
@@ -152,7 +152,7 @@ yourself using the configuration menus:
 .. code-block:: console
 
    $ export CT_PREFIX=/Volumes/CrossToolNG
-   $ ct-ng menuconfig
+   $ ct-ng oldconfig
 
 Verifying the Configuration of the Toolchain
 ============================================

--- a/scripts/cross_compiler/i586.config
+++ b/scripts/cross_compiler/i586.config
@@ -2,19 +2,6 @@
 # Automatically generated file; DO NOT EDIT.
 # Crosstool-NG Configuration
 #
-CT_CONFIGURE_has_wget=y
-CT_CONFIGURE_has_curl=y
-CT_CONFIGURE_has_stat_flavor_GNU=y
-CT_CONFIGURE_has_make_3_81_or_newer=y
-CT_CONFIGURE_has_libtool_2_4_or_newer=y
-CT_CONFIGURE_has_libtoolize_2_4_or_newer=y
-CT_CONFIGURE_has_autoconf_2_63_or_newer=y
-CT_CONFIGURE_has_autoreconf_2_63_or_newer=y
-CT_CONFIGURE_has_automake_1_15_or_newer=y
-CT_CONFIGURE_has_gnu_m4_1_4_12_or_newer=y
-CT_CONFIGURE_has_cvs=y
-CT_CONFIGURE_has_svn=y
-CT_CONFIGURE_has_git=y
 CT_MODULES=y
 
 #
@@ -24,7 +11,7 @@ CT_MODULES=y
 #
 # crosstool-NG behavior
 #
-# CT_OBSOLETE is not set
+CT_OBSOLETE=y
 # CT_EXPERIMENTAL is not set
 # CT_DEBUG_CT is not set
 
@@ -158,6 +145,10 @@ CT_TARGET_CFLAGS="-m32"
 CT_TARGET_LDFLAGS=""
 CT_ARCH_FLOAT=""
 
+#
+# x86 other options
+#
+# CT_ARCH_X86_IAMCU is not set
 
 #
 # Toolchain options
@@ -225,6 +216,9 @@ CT_BINUTILS_VERSION="2.28"
 CT_BINUTILS_V_2_28=y
 # CT_BINUTILS_V_2_27 is not set
 # CT_BINUTILS_V_2_26 is not set
+# CT_BINUTILS_V_2_25_1 is not set
+# CT_BINUTILS_V_2_24 is not set
+# CT_BINUTILS_V_2_23_2 is not set
 CT_BINUTILS_2_27_or_later=y
 CT_BINUTILS_2_26_or_later=y
 CT_BINUTILS_2_25_1_or_later=y
@@ -329,6 +323,7 @@ CT_CC_GCC_VERSION="6.3.0"
 CT_CC_GCC_V_6_3_0=y
 # CT_CC_GCC_V_5_4_0 is not set
 # CT_CC_GCC_V_4_9_4 is not set
+# CT_CC_GCC_V_4_8_5 is not set
 CT_CC_GCC_4_8_or_later=y
 CT_CC_GCC_4_9_or_later=y
 CT_CC_GCC_5_or_later=y
@@ -408,10 +403,30 @@ CT_GDB_CROSS_EXTRA_CONFIG_ARRAY=""
 #
 # gdb version
 #
-CT_GDB_VERSION="7.12.1"
-CT_GDB_V_7_12_1=y
-# CT_GDB_V_7_11_1 is not set
-CT_GDB_7_12_or_later=y
+CT_GDB_VERSION="7.11.1"
+# CT_DEBUG_GDB_SHOW_LINARO is not set
+# CT_GDB_V_7_12_1 is not set
+CT_GDB_V_7_11_1=y
+# CT_GDB_V_7_10_1 is not set
+# CT_GDB_V_7_10 is not set
+# CT_GDB_V_7_9_1 is not set
+# CT_GDB_V_7_9 is not set
+# CT_GDB_V_7_8_2 is not set
+# CT_GDB_V_7_8_1 is not set
+# CT_GDB_V_7_8 is not set
+# CT_GDB_V_7_7_1 is not set
+# CT_GDB_V_7_7 is not set
+# CT_GDB_V_7_6_1 is not set
+# CT_GDB_V_7_5_1 is not set
+# CT_GDB_V_7_4_1 is not set
+# CT_GDB_V_7_4 is not set
+# CT_GDB_V_7_3_1 is not set
+# CT_GDB_V_7_3a is not set
+# CT_GDB_V_7_2a is not set
+# CT_GDB_V_7_1a is not set
+# CT_GDB_V_7_0_1a is not set
+# CT_GDB_V_7_0a is not set
+# CT_GDB_V_6_8a is not set
 CT_GDB_7_2_or_later=y
 CT_GDB_7_0_or_later=y
 CT_GDB_HAS_PKGVERSION_BUGURL=y
@@ -446,23 +461,50 @@ CT_ZLIB_VERSION="1.2.11"
 CT_LIBICONV_V_1_15=y
 # CT_LIBICONV_V_1_14 is not set
 CT_LIBICONV_VERSION="1.15"
-CT_GETTEXT_V_0_19_8_1=y
-CT_GETTEXT_VERSION="0.19.8.1"
+# CT_GETTEXT_V_0_19_8_1 is not set
+CT_GETTEXT_V_0_19_7=y
+CT_GETTEXT_VERSION="0.19.7"
 CT_GMP_V_6_1_2=y
+# CT_GMP_V_6_1_0 is not set
+# CT_GMP_V_6_0_0 is not set
+# CT_GMP_V_5_1_3 is not set
+# CT_GMP_V_5_1_1 is not set
+# CT_GMP_V_5_0_2 is not set
+# CT_GMP_V_5_0_1 is not set
+# CT_GMP_V_4_3_2 is not set
+# CT_GMP_V_4_3_1 is not set
+# CT_GMP_V_4_3_0 is not set
 CT_GMP_5_0_2_or_later=y
 CT_GMP_VERSION="6.1.2"
 CT_MPFR_V_3_1_5=y
+# CT_MPFR_V_3_1_3 is not set
+# CT_MPFR_V_3_1_2 is not set
+# CT_MPFR_V_3_1_0 is not set
+# CT_MPFR_V_3_0_1 is not set
+# CT_MPFR_V_3_0_0 is not set
+# CT_MPFR_V_2_4_2 is not set
+# CT_MPFR_V_2_4_1 is not set
+# CT_MPFR_V_2_4_0 is not set
 CT_MPFR_VERSION="3.1.5"
 CT_ISL_V_0_16_1=y
 # CT_ISL_V_0_15 is not set
+# CT_ISL_V_0_14 is not set
 CT_ISL_V_0_16_or_later=y
 CT_ISL_V_0_15_or_later=y
 CT_ISL_V_0_14_or_later=y
 CT_ISL_V_0_12_or_later=y
 CT_ISL_VERSION="0.16.1"
 CT_MPC_V_1_0_3=y
+# CT_MPC_V_1_0_2 is not set
+# CT_MPC_V_1_0_1 is not set
+# CT_MPC_V_1_0 is not set
+# CT_MPC_V_0_9 is not set
+# CT_MPC_V_0_8_2 is not set
+# CT_MPC_V_0_8_1 is not set
+# CT_MPC_V_0_7 is not set
 CT_MPC_VERSION="1.0.3"
 CT_EXPAT_V_2_2_0=y
+# CT_EXPAT_V_2_1_1 is not set
 CT_EXPAT_VERSION="2.2.0"
 CT_NCURSES_V_6_0=y
 CT_NCURSES_VERSION="6.0"
@@ -487,5 +529,7 @@ CT_NCURSES_TARGET_FALLBACKS=""
 # CT_COMP_TOOLS_libtool is not set
 CT_COMP_TOOLS_m4=y
 CT_M4_V_1_4_18=y
+# CT_M4_V_1_4_17 is not set
+# CT_M4_V_1_4_13 is not set
 CT_M4_VERSION="1.4.18"
 # CT_COMP_TOOLS_make is not set

--- a/scripts/cross_compiler/iamcu.config
+++ b/scripts/cross_compiler/iamcu.config
@@ -2,19 +2,6 @@
 # Automatically generated file; DO NOT EDIT.
 # Crosstool-NG Configuration
 #
-CT_CONFIGURE_has_wget=y
-CT_CONFIGURE_has_curl=y
-CT_CONFIGURE_has_stat_flavor_GNU=y
-CT_CONFIGURE_has_make_3_81_or_newer=y
-CT_CONFIGURE_has_libtool_2_4_or_newer=y
-CT_CONFIGURE_has_libtoolize_2_4_or_newer=y
-CT_CONFIGURE_has_autoconf_2_63_or_newer=y
-CT_CONFIGURE_has_autoreconf_2_63_or_newer=y
-CT_CONFIGURE_has_automake_1_15_or_newer=y
-CT_CONFIGURE_has_gnu_m4_1_4_12_or_newer=y
-CT_CONFIGURE_has_cvs=y
-CT_CONFIGURE_has_svn=y
-CT_CONFIGURE_has_git=y
 CT_MODULES=y
 
 #
@@ -24,7 +11,7 @@ CT_MODULES=y
 #
 # crosstool-NG behavior
 #
-# CT_OBSOLETE is not set
+CT_OBSOLETE=y
 # CT_EXPERIMENTAL is not set
 # CT_DEBUG_CT is not set
 
@@ -229,6 +216,9 @@ CT_BINUTILS_VERSION="2.28"
 CT_BINUTILS_V_2_28=y
 # CT_BINUTILS_V_2_27 is not set
 # CT_BINUTILS_V_2_26 is not set
+# CT_BINUTILS_V_2_25_1 is not set
+# CT_BINUTILS_V_2_24 is not set
+# CT_BINUTILS_V_2_23_2 is not set
 CT_BINUTILS_2_27_or_later=y
 CT_BINUTILS_2_26_or_later=y
 CT_BINUTILS_2_25_1_or_later=y
@@ -333,6 +323,7 @@ CT_CC_GCC_VERSION="6.3.0"
 CT_CC_GCC_V_6_3_0=y
 # CT_CC_GCC_V_5_4_0 is not set
 # CT_CC_GCC_V_4_9_4 is not set
+# CT_CC_GCC_V_4_8_5 is not set
 CT_CC_GCC_4_8_or_later=y
 CT_CC_GCC_4_9_or_later=y
 CT_CC_GCC_5_or_later=y
@@ -412,10 +403,30 @@ CT_GDB_CROSS_EXTRA_CONFIG_ARRAY=""
 #
 # gdb version
 #
-CT_GDB_VERSION="7.12.1"
-CT_GDB_V_7_12_1=y
-# CT_GDB_V_7_11_1 is not set
-CT_GDB_7_12_or_later=y
+CT_GDB_VERSION="7.11.1"
+# CT_DEBUG_GDB_SHOW_LINARO is not set
+# CT_GDB_V_7_12_1 is not set
+CT_GDB_V_7_11_1=y
+# CT_GDB_V_7_10_1 is not set
+# CT_GDB_V_7_10 is not set
+# CT_GDB_V_7_9_1 is not set
+# CT_GDB_V_7_9 is not set
+# CT_GDB_V_7_8_2 is not set
+# CT_GDB_V_7_8_1 is not set
+# CT_GDB_V_7_8 is not set
+# CT_GDB_V_7_7_1 is not set
+# CT_GDB_V_7_7 is not set
+# CT_GDB_V_7_6_1 is not set
+# CT_GDB_V_7_5_1 is not set
+# CT_GDB_V_7_4_1 is not set
+# CT_GDB_V_7_4 is not set
+# CT_GDB_V_7_3_1 is not set
+# CT_GDB_V_7_3a is not set
+# CT_GDB_V_7_2a is not set
+# CT_GDB_V_7_1a is not set
+# CT_GDB_V_7_0_1a is not set
+# CT_GDB_V_7_0a is not set
+# CT_GDB_V_6_8a is not set
 CT_GDB_7_2_or_later=y
 CT_GDB_7_0_or_later=y
 CT_GDB_HAS_PKGVERSION_BUGURL=y
@@ -450,23 +461,50 @@ CT_ZLIB_VERSION="1.2.11"
 CT_LIBICONV_V_1_15=y
 # CT_LIBICONV_V_1_14 is not set
 CT_LIBICONV_VERSION="1.15"
-CT_GETTEXT_V_0_19_8_1=y
-CT_GETTEXT_VERSION="0.19.8.1"
+# CT_GETTEXT_V_0_19_8_1 is not set
+CT_GETTEXT_V_0_19_7=y
+CT_GETTEXT_VERSION="0.19.7"
 CT_GMP_V_6_1_2=y
+# CT_GMP_V_6_1_0 is not set
+# CT_GMP_V_6_0_0 is not set
+# CT_GMP_V_5_1_3 is not set
+# CT_GMP_V_5_1_1 is not set
+# CT_GMP_V_5_0_2 is not set
+# CT_GMP_V_5_0_1 is not set
+# CT_GMP_V_4_3_2 is not set
+# CT_GMP_V_4_3_1 is not set
+# CT_GMP_V_4_3_0 is not set
 CT_GMP_5_0_2_or_later=y
 CT_GMP_VERSION="6.1.2"
 CT_MPFR_V_3_1_5=y
+# CT_MPFR_V_3_1_3 is not set
+# CT_MPFR_V_3_1_2 is not set
+# CT_MPFR_V_3_1_0 is not set
+# CT_MPFR_V_3_0_1 is not set
+# CT_MPFR_V_3_0_0 is not set
+# CT_MPFR_V_2_4_2 is not set
+# CT_MPFR_V_2_4_1 is not set
+# CT_MPFR_V_2_4_0 is not set
 CT_MPFR_VERSION="3.1.5"
 CT_ISL_V_0_16_1=y
 # CT_ISL_V_0_15 is not set
+# CT_ISL_V_0_14 is not set
 CT_ISL_V_0_16_or_later=y
 CT_ISL_V_0_15_or_later=y
 CT_ISL_V_0_14_or_later=y
 CT_ISL_V_0_12_or_later=y
 CT_ISL_VERSION="0.16.1"
 CT_MPC_V_1_0_3=y
+# CT_MPC_V_1_0_2 is not set
+# CT_MPC_V_1_0_1 is not set
+# CT_MPC_V_1_0 is not set
+# CT_MPC_V_0_9 is not set
+# CT_MPC_V_0_8_2 is not set
+# CT_MPC_V_0_8_1 is not set
+# CT_MPC_V_0_7 is not set
 CT_MPC_VERSION="1.0.3"
 CT_EXPAT_V_2_2_0=y
+# CT_EXPAT_V_2_1_1 is not set
 CT_EXPAT_VERSION="2.2.0"
 CT_NCURSES_V_6_0=y
 CT_NCURSES_VERSION="6.0"
@@ -491,5 +529,7 @@ CT_NCURSES_TARGET_FALLBACKS=""
 # CT_COMP_TOOLS_libtool is not set
 CT_COMP_TOOLS_m4=y
 CT_M4_V_1_4_18=y
+# CT_M4_V_1_4_17 is not set
+# CT_M4_V_1_4_13 is not set
 CT_M4_VERSION="1.4.18"
 # CT_COMP_TOOLS_make is not set

--- a/scripts/cross_compiler/nios2.config
+++ b/scripts/cross_compiler/nios2.config
@@ -2,19 +2,6 @@
 # Automatically generated file; DO NOT EDIT.
 # Crosstool-NG Configuration
 #
-CT_CONFIGURE_has_wget=y
-CT_CONFIGURE_has_curl=y
-CT_CONFIGURE_has_stat_flavor_GNU=y
-CT_CONFIGURE_has_make_3_81_or_newer=y
-CT_CONFIGURE_has_libtool_2_4_or_newer=y
-CT_CONFIGURE_has_libtoolize_2_4_or_newer=y
-CT_CONFIGURE_has_autoconf_2_63_or_newer=y
-CT_CONFIGURE_has_autoreconf_2_63_or_newer=y
-CT_CONFIGURE_has_automake_1_15_or_newer=y
-CT_CONFIGURE_has_gnu_m4_1_4_12_or_newer=y
-CT_CONFIGURE_has_cvs=y
-CT_CONFIGURE_has_svn=y
-CT_CONFIGURE_has_git=y
 CT_MODULES=y
 
 #
@@ -24,7 +11,7 @@ CT_MODULES=y
 #
 # crosstool-NG behavior
 #
-# CT_OBSOLETE is not set
+CT_OBSOLETE=y
 # CT_EXPERIMENTAL is not set
 # CT_DEBUG_CT is not set
 
@@ -220,6 +207,9 @@ CT_BINUTILS_VERSION="2.28"
 CT_BINUTILS_V_2_28=y
 # CT_BINUTILS_V_2_27 is not set
 # CT_BINUTILS_V_2_26 is not set
+# CT_BINUTILS_V_2_25_1 is not set
+# CT_BINUTILS_V_2_24 is not set
+# CT_BINUTILS_V_2_23_2 is not set
 CT_BINUTILS_2_27_or_later=y
 CT_BINUTILS_2_26_or_later=y
 CT_BINUTILS_2_25_1_or_later=y
@@ -319,6 +309,7 @@ CT_CC_GCC_VERSION="6.3.0"
 CT_CC_GCC_V_6_3_0=y
 # CT_CC_GCC_V_5_4_0 is not set
 # CT_CC_GCC_V_4_9_4 is not set
+# CT_CC_GCC_V_4_8_5 is not set
 CT_CC_GCC_4_8_or_later=y
 CT_CC_GCC_4_9_or_later=y
 CT_CC_GCC_5_or_later=y
@@ -397,10 +388,30 @@ CT_GDB_CROSS_EXTRA_CONFIG_ARRAY=""
 #
 # gdb version
 #
-CT_GDB_VERSION="7.12.1"
-CT_GDB_V_7_12_1=y
-# CT_GDB_V_7_11_1 is not set
-CT_GDB_7_12_or_later=y
+CT_GDB_VERSION="7.11.1"
+# CT_DEBUG_GDB_SHOW_LINARO is not set
+# CT_GDB_V_7_12_1 is not set
+CT_GDB_V_7_11_1=y
+# CT_GDB_V_7_10_1 is not set
+# CT_GDB_V_7_10 is not set
+# CT_GDB_V_7_9_1 is not set
+# CT_GDB_V_7_9 is not set
+# CT_GDB_V_7_8_2 is not set
+# CT_GDB_V_7_8_1 is not set
+# CT_GDB_V_7_8 is not set
+# CT_GDB_V_7_7_1 is not set
+# CT_GDB_V_7_7 is not set
+# CT_GDB_V_7_6_1 is not set
+# CT_GDB_V_7_5_1 is not set
+# CT_GDB_V_7_4_1 is not set
+# CT_GDB_V_7_4 is not set
+# CT_GDB_V_7_3_1 is not set
+# CT_GDB_V_7_3a is not set
+# CT_GDB_V_7_2a is not set
+# CT_GDB_V_7_1a is not set
+# CT_GDB_V_7_0_1a is not set
+# CT_GDB_V_7_0a is not set
+# CT_GDB_V_6_8a is not set
 CT_GDB_7_2_or_later=y
 CT_GDB_7_0_or_later=y
 CT_GDB_HAS_PKGVERSION_BUGURL=y
@@ -435,23 +446,50 @@ CT_ZLIB_VERSION="1.2.11"
 CT_LIBICONV_V_1_15=y
 # CT_LIBICONV_V_1_14 is not set
 CT_LIBICONV_VERSION="1.15"
-CT_GETTEXT_V_0_19_8_1=y
-CT_GETTEXT_VERSION="0.19.8.1"
+# CT_GETTEXT_V_0_19_8_1 is not set
+CT_GETTEXT_V_0_19_7=y
+CT_GETTEXT_VERSION="0.19.7"
 CT_GMP_V_6_1_2=y
+# CT_GMP_V_6_1_0 is not set
+# CT_GMP_V_6_0_0 is not set
+# CT_GMP_V_5_1_3 is not set
+# CT_GMP_V_5_1_1 is not set
+# CT_GMP_V_5_0_2 is not set
+# CT_GMP_V_5_0_1 is not set
+# CT_GMP_V_4_3_2 is not set
+# CT_GMP_V_4_3_1 is not set
+# CT_GMP_V_4_3_0 is not set
 CT_GMP_5_0_2_or_later=y
 CT_GMP_VERSION="6.1.2"
 CT_MPFR_V_3_1_5=y
+# CT_MPFR_V_3_1_3 is not set
+# CT_MPFR_V_3_1_2 is not set
+# CT_MPFR_V_3_1_0 is not set
+# CT_MPFR_V_3_0_1 is not set
+# CT_MPFR_V_3_0_0 is not set
+# CT_MPFR_V_2_4_2 is not set
+# CT_MPFR_V_2_4_1 is not set
+# CT_MPFR_V_2_4_0 is not set
 CT_MPFR_VERSION="3.1.5"
 CT_ISL_V_0_16_1=y
 # CT_ISL_V_0_15 is not set
+# CT_ISL_V_0_14 is not set
 CT_ISL_V_0_16_or_later=y
 CT_ISL_V_0_15_or_later=y
 CT_ISL_V_0_14_or_later=y
 CT_ISL_V_0_12_or_later=y
 CT_ISL_VERSION="0.16.1"
 CT_MPC_V_1_0_3=y
+# CT_MPC_V_1_0_2 is not set
+# CT_MPC_V_1_0_1 is not set
+# CT_MPC_V_1_0 is not set
+# CT_MPC_V_0_9 is not set
+# CT_MPC_V_0_8_2 is not set
+# CT_MPC_V_0_8_1 is not set
+# CT_MPC_V_0_7 is not set
 CT_MPC_VERSION="1.0.3"
 CT_EXPAT_V_2_2_0=y
+# CT_EXPAT_V_2_1_1 is not set
 CT_EXPAT_VERSION="2.2.0"
 CT_NCURSES_V_6_0=y
 CT_NCURSES_VERSION="6.0"
@@ -476,8 +514,13 @@ CT_NCURSES_TARGET_FALLBACKS=""
 # CT_COMP_TOOLS_libtool is not set
 CT_COMP_TOOLS_m4=y
 CT_M4_V_1_4_18=y
+# CT_M4_V_1_4_17 is not set
+# CT_M4_V_1_4_13 is not set
 CT_M4_VERSION="1.4.18"
 CT_COMP_TOOLS_make=y
 CT_MAKE_V_4_2_1=y
+# CT_MAKE_V_4_1 is not set
+# CT_MAKE_V_4_0 is not set
+# CT_MAKE_V_3_81 is not set
 # CT_MAKE_GMAKE_SYMLINK is not set
 CT_MAKE_VERSION="4.2.1"

